inspirehep/demosite/data/collection.json

```diff
@@ -0,0 +1,82 @@
+[
```

Comment:
Not sure, it is a fixture that will have to be loaded once into the DB (and also needs to be loaded into the demo site for developers). Dunno what would be a good place.